helm/README.md

```diff
@@ -0,0 +1,7 @@
+This project is a component of the [Operator Framework](https://github.com/operator-framework), an open source toolkit to manage Kubernetes native applications, called Operators, in an effective, automated, and scalable way. Read more in the [introduction blog post](https://coreos.com/blog/introducing-operator-framework).
+
+# Operator SDK Samples - Helm
+This repo contains samples of operators powered by Helm built using the [operator-sdk][operator_sdk]. To learn more about creating an operator that leverages Helm, check out the [Helm user guide][helm_user_guide].
```

Comment:
Nit: This directory maybe, instead of repo? As this repo contains more than just helm samples.